zebrad/tests/acceptance.rs

```diff
@@ -284,7 +284,23 @@ fn persistent_mode() -> Result<()> {
 #[test]
 fn ephemeral_mode() -> Result<()> {
```

Comment:
We still want to keep the old version that tests `ConfigMode::Ephemeral` when its configured correctly, can you add back the old version of the test as a separate test and then rename this version to `fn misconfigured_ephemeral_mode()`?

Reply:
Thanks, looks good!